### ISSUE TITLE
Getting ShikiError with `Markdown`

### ISSUE THREAD

**Issue by mohit2152sharma** [BearToCode/carta#56]
Hi, 
I am using `Markdown` (https://beartocode.github.io/carta/api/core#markdown-options) to render a markdown. The markdown includes code. I have created a component with `Markdown` and then I am using it in a parent component. Following is my `Markdown` component:

```svelte
<script lang='ts'>
	import { Markdown, Carta } from 'carta-md';
	import '$lib/styles/github.scss';
	import DOMPurify from 'isomorphic-dompurify';
	import { code } from '@cartamd/plugin-code';
	let carta = new Carta({
		sanitizer: DOMPurify.sanitize,
		extensions: [code()]
	});
	export let noteText: string;
</script>

<Markdown bind:value={noteText} {carta} theme="github" />
```
The above throws the following error.

```
Uncaught (in promise) ShikiError: Language `python` not found, you may need to load it first
    at Object.getLanguage (chunk-64SPV7BA.js?v=a4fccd20:8133:13)
    at codeToTokensBase (chunk-64SPV7BA.js?v=a4fccd20:4713:29)
    at chunk-64SPV7BA.js?v=a4fccd20:4848:62
    at Array.map (<anonymous>)
    at codeToTokensWithThemes (chunk-64SPV7BA.js?v=a4fccd20:4848:51)
    at codeToTokens (chunk-64SPV7BA.js?v=a4fccd20:4916:25)
    at codeToHast (chunk-64SPV7BA.js?v=a4fccd20:5144:50)
    at Object.codeToHast (chunk-64SPV7BA.js?v=a4fccd20:8184:37)
    at @cartamd_plugin-code.js?v=a4fccd20:81:38
    at overload (chunk-5AD3OJQS.js?v=a4fccd20:195:12)
```
I have tried with other languages as well, but I get the same error. However, when I used `MarkdownEditor`, it is able to render code markdown properly without in both preview and editor tab. Not sure, If I am using it correctly.

**Comment by BearToCode** [comment]
Hello, I can't reproduce the issue for now. [This](https://www.sveltelab.dev/n3ehpcnudifkdd7?files=.%2Fsrc%2Froutes%2F%2Bpage.svelte) is what I tried so far, and it seems to be working. What value of `noteText` specifically causes the issue?

**Comment by mohit2152sharma** [comment]
Okay, so I have narrowed it down to the changes which are leading to this issue. 

First, I am using `MarkdownEditor`, the `value` field of the component gets prepoulated with a default value (the idea is to fetch and show saved values by user). This renders the markdown fine. For example for the following string: 

```
noteText =  "testing python string \n\n```python\ndef f():\n\tprint('hello world really with really long long long long long long string')\n```"
```

The above string renders fine, in both `MarkdownEditor` and `Markdown` component.

I have a Save button, which will pass any edits to the server to save to the database. When I click on Save, the following string gets passed down to the server: 

```
 "testing python string \r\n\r\n```python\r\ndef f():\r\n\tprint('hello world really with really long long long long long long string')\r\n```"
```

Notice, the `\r` that gets added to the `\n` characters. These got added automatically, without making me any changes. Now when I try to render this string in `Markdown`, I get the original error (mentioned in this post).
